leap/multistep/__init__.py

```diff
@@ -219,7 +222,7 @@ class AdamsBashforthMethodBuilder(MethodBuilder):
     """
 
     def __init__(self, component_id, function_family=None, state_filter_name=None,
-            hist_length=None, static_dt=False, order=None):
+            hist_length=None, static_dt=False, order=None, _extra_bootstrap=False):
```

Comment:
I'm not liking this `_extra_bootstrap` thing, mostly because it's ill-defined.

- What happens with the extra history after bootstrap?
- What are its semantics in the explicit case?
- The justification for it seems really weak and mostly focused on debugging?

Reply:
This is an option that was previously in place to pass a certain test that compared multi-rate IMEX to single-rate implicit. Given that that's not what we're talking about here, this is an artifact of outdated development and needs to be removed. I'm on it.

Reply:
This option and its artifacts are removed by ae9d04f.